hack/crossplane/apis/sql/v1beta20201101preview/structure.txt

```diff
@@ -147,7 +147,7 @@ github.com/Azure/azure-service-operator/hack/crossplane/apis/sql/v1beta20201101p
     │       ├── ResourceGroupNameRef: *v1alpha1.Reference
     │       ├── ResourceGroupNameSelector: *v1alpha1.Selector
     │       ├── RestorableDroppedDatabaseId: *string
-    │       ├── RestorePointInTime: Validated<*string> (0 rules)
+    │       ├── RestorePointInTime: *string
```

Comment:
Note that these changes (from `Validated<*string> (0 rules)` to `*string`) don't have any structural impact to the CRDs and so are nonbreaking.